src/c2pa/c2pa.py

```diff
@@ -1242,7 +1226,14 @@ def initialized(self) -> bool:
 
 
 class Reader:
-    """High-level wrapper for C2PA Reader operations."""
+    """High-level wrapper for C2PA Reader operations.
+    
+    Example:
+        ```
+        with Reader(path) as reader:
```

Comment:
Technically, we would prefer if people use the stream API instead.
The stream API is like this: `with Reader("image/jpeg", output) as reader:`, where output is either an in-memory stream, or an already oepned file.

Reply:
Changed the example code accordingly.